docs/policygenerator-reference.yaml

```diff
@@ -127,6 +129,9 @@ policies:
     # Optional. Determines the policy controller behavior when comparing the manifest to objects on
     # the cluster ("musthave",  "mustonlyhave", or "mustnothave"). Defaults to "musthave".
     complianceType: "musthave"
+    # Optional. Overrides complianceType when comparing the manifest's metadata section to objects on
+    # the cluster ("musthave",  "mustonlyhave"). Default is unset to not override complianceType for metadata.
+    metadataComplianceType: ""
```

Comment:
Could you please also document this under `policyDefaults`?

Reply:
I did it as current PR  because I'm not sure if we also want `metadataComplianceType` under `policyDefaults`, or just under `policies` and `manifests` as `complianceType`, since `metadataComplianceType` is an enhancement of `complianceType`. If confirmed, I can update it here.